geospaas_processing/copiers.py

```diff
@@ -30,9 +30,15 @@ def write_flag_file(type_in_flag_file, source_path, dataset, destination_filenam
         writes the flag file in the case of request for it. ".flag" is the extension for flag file.
         """
         with open(destination_filename + ".flag", "w") as flag_file:
-            flag_file.write(f"type: {type_in_flag_file}" + os.linesep)
+            string_to_write = f"type: {type_in_flag_file}" + os.linesep
```

Comment:
You already use f-strings, so you don't need to concatenate the line separator:
`f"type: {type_in_flag_file}{os.linesep}"`

Reply:
Done in 4a22268